config/routes.rb

```diff
@@ -5,14 +5,14 @@
   delete '/logout', to: 'sessions#destroy', as: 'logout'
 
   resources :categories, only: [:show, :index]
-  resources :items
-  get "cart_items/create"
+  resources :projects
+  get "cart_projects/create"
 
   post '/carts', to: 'carts#create', as: 'carts'
   get '/cart', to: 'carts#showcart', as: 'showcart'
   post '/cart', to: 'carts#checkout_cart', as: 'checkout_cart'
-  delete '/cart', to: 'carts#delete_item', as: 'cart'
-  put '/cart', to: 'carts#update_item_quantity', as: 'update_item_quantity'
+  delete '/cart', to: 'carts#delete_project', as: 'cart'
+  put '/cart', to: 'carts#update_project_quantity', as: 'update_project_quantity'
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.